### ISSUE TITLE
Integration freezing?

### ISSUE THREAD

**Issue by JayMcG** [wernerhp/ha.integration.load_shedding#71]
Hi,

I've noticed a few times that my LS integration, for lack of a better word, freezes. I then have to reload the integration and all works perfectly again. Not sure if anyone else has experienced this?

**Comment by wernerhp** [comment]
Yeah, am aware of this.  Not sure what the issue is though.  There's something wrong with the schedule's coordinator.  Even though it calls the API every day to get the latest schedule, for some reason it doesn't pull through right and so every day you end up with 6...5...4...3...2...1 and finally 0 days of forecast.  The current workaround is an automation to reload the integration once daily.

https://github.com/wernerhp/ha.integration.load_shedding/issues/70#issuecomment-1568086066

**Comment by wernerhp** [comment]
The issue kight be fixed in https://github.com/wernerhp/ha.integration.load_shedding/releases/tag/v1.5.0-beta.2 . Need to test.

**Comment by wernerhp** [comment]
👋 A fix for this issue is included in the **v1.7.0-dev.1** pre-release ([#114](https://github.com/wernerhp/ha.integration.load_shedding/pull/114)), now available via HACS for testing.

To try it:
1. In HACS, open **Load Shedding** → ⋮ menu → **Redownload**.
2. Enable **Show beta versions**.
3. Select **v1.7.0-dev.1** and download, then restart Home Assistant.

If you're comfortable with it, please report back here whether it resolves the problem for you — that feedback helps confirm the fix before a stable release. 🙏

Release notes: https://github.com/wernerhp/ha.integration.load_shedding/releases/tag/v1.7.0-dev.1
